Modules/Core/Transform/include/itkKernelTransform.hxx

```diff
@@ -508,6 +508,43 @@ KernelTransform<TParametersValueType, VDimension>::GetFixedParameters() const ->
 }
 
 
+template <typename TParametersValueType, unsigned int VDimension>
+bool
+KernelTransform<TParametersValueType, VDimension>::GetInverse(Self * inverseTransform) const
+{
+  if (!inverseTransform)
+  {
+    return false;
+  }
+
+  auto deepCopyLandmarks = [](const typename PointSetType::ConstPointer source,
+                              typename PointSetType::Pointer            destination) {
+    typename PointSetType::PointsContainer::ConstIterator sourceIt = source->GetPoints()->Begin();
+
+    typename PointSetType::PointIdentifier i{ 0 };
+    while (sourceIt != source->GetPoints()->End())
+    {
+      destination->SetPoint(i, sourceIt->Value());
+      ++i;
+      ++sourceIt;
+    }
+  };
+
+  // make a deep copy of the source and target landmarks
+  typename PointSetType::Pointer sourceLandmarks = PointSetType::New();
+  typename PointSetType::Pointer targetLandmarks = PointSetType::New();
+  deepCopyLandmarks(this->GetSourceLandmarks(), sourceLandmarks);
+  deepCopyLandmarks(this->GetTargetLandmarks(), targetLandmarks);
+
+  // inversion comes from reversing the landmarks
+  inverseTransform->SetSourceLandmarks(targetLandmarks);
+  inverseTransform->SetTargetLandmarks(sourceLandmarks);
+  inverseTransform->SetStiffness(this->GetStiffness());
```

Comment:
I think it's OK for a class to directly access its own data members, without calling Get and Set. So more like:

```cpp
  inverseTransform->m_SourceLandmarks = targetLandmarks;
  inverseTransform->m_TargetLandmarks = sourceLandmarks;
  inverseTransform->m_Stiffness = m_Stiffness;
```

Right?